smalltalksrc/VMMaker/DruidJIT.class.st

```diff
@@ -7282,8 +7166,7 @@ DruidJIT >> gen_StoreAndPopReceiverVariableBytecode0 [
 	jump3 jmpTarget: currentBlock.
 	jump6 jmpTarget: currentBlock.
 	jump1 jmpTarget: currentBlock.
-	self deoptimize.
-	deadCode := false.
+	self MoveR: t1 M64: 8 r: t0.
```

Comment:
??? Is it right?

Reply:
I think so, no? Why should we have two deoptimizes?

Reply:
Ahh yes! This is good... strange the old version...